### PR TITLE
BulkUpdate repos, migrate to parcel

### PR DESCRIPTION
**Pull request by YuriiHoliuk**
- add updateRepos script
- migrate layout repositories from gulp to parcel
- `npm audit fix` in layout repositories
- make migrate command async

### CODE REVIEW

File: mate-scripts/.env.sample [hunk 1]
```diff
@@ -0,0 +1 @@
+GITHUB_TOKEN=your token
```
Comment (ivadimko) [inline mate-scripts/.env.sample:1]: does it mean students should add their tokens? Or it's just for dev purposes

File: mate-scripts/src/localScripts/lastCreatedPRs.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+[
```
Comment (ivadimko) [inline mate-scripts/src/localScripts/lastCreatedPRs.json:1]: do we need this file to be stored in the repo? it looks like dynamic data

File: mate-scripts/src/localScripts/repositoriesData.json [hunk 3]
```diff
@@ -0,0 +1,173 @@
+{
```
Comment (ivadimko) [inline mate-scripts/src/localScripts/repositoriesData.json:1]: looks like dynamic data. do we need to store it?

File: mate-scripts/src/localScripts/updateRepos/ReposUpdater.ts [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        owner: orgName,
+        repo,
+        pull_number: pullNumber,
+        mergeMethod,
+      });
+
+      await execInRepo(`git push --delete origin ${prBranch}`);
+    }
+
+    console.log('Done with ', repo);
```
Comment (ivadimko) [inline mate-scripts/src/localScripts/updateRepos/ReposUpdater.ts:120]: image

File: mate-scripts/.env.sample (hunk 1 above)
Comment (YuriiHoliuk) [reply mate-scripts/.env.sample:1]: no, local script not published vie npm it is only for dev local usage

File: mate-scripts/src/localScripts/repositoriesData.json (hunk 3 above)
Comment (YuriiHoliuk) [reply mate-scripts/src/localScripts/repositoriesData.json:1]: not fully dynamic, better to leave as is

File: mate-scripts/src/localScripts/updateRepos/ReposUpdater.ts (hunk 4 above)
Comment (YuriiHoliuk) [reply mate-scripts/src/localScripts/updateRepos/ReposUpdater.ts:120]: Як дійдуть руки, зроблю, щоб виглядало простіше, можна винести в методи mergePR, cloneRepo, але я і так занадто заморочився))

**Pull request merged**